__tests__/taskRequests/taskRequest.test.js

```diff
@@ -39,16 +41,31 @@ describe('Task Requests', () => {
             'Access-Control-Allow-Headers': 'Content-Type, Authorization',
           },
         });
+      } else if (
+        request.url() ===
+        `${API_BASE_URL}/taskRequests?size=20&q=status%3Aapproved++sort%3Acreated-asc&dev=true`
+      ) {
+        const list = [];
+        for (let i = 0; i < 20; i++) {
+          list.push(fetchedTaskRequests[0]);
+        }
+        request.respond({
+          status: 200,
+          contentType: 'application/json',
+          body: JSON.stringify({
+            data: list,
+            next: '/taskRequests?size=20&q=status%3Aapproved++sort%3Acreated-asc&dev=true',
+          }),
+        });
       } else {
         request.continue();
       }
     });
-
     await page.goto(`${SITE_URL}/taskRequests`);
     await page.waitForNetworkIdle();
   });
 
-  afterAll(async () => {
+  afterEach(async () => {
```

Comment:
Is it not a good idea to close browser after all the tests instead of each test

Reply:
removed browser.close() and replaced it with page.close()